src/actix_web/http_compat.rs

```diff
@@ -0,0 +1,29 @@
+#![allow(clippy::unwrap_used)]
```

Comment:
Get rid of this and do proper error handling with `?`, otherwise the code can crash at runtime. I realize that there should be no errors as all values are already validated, but yet the code should be written cleanly.

Reply:
Or simply use `.expect()`

Reply:
@Nutomic  Since [the author has been inactive recently](https://github.com/pbzweihander), maybe merge it first and then make changes?
I believe using `.expect()` would be appropriate.

Reply:
Could also just edit this PR, or branch off it directly.

Reply:
(is silently in favor of editing it; I'm using this branch to currently use this project as I'm using a newer version of Axum)